cfgov/paying_for_college/documents.py

```diff
@@ -0,0 +1,35 @@
+from django_elasticsearch_dsl import Document, fields
+from django_elasticsearch_dsl.registries import registry
+
+from paying_for_college.models import School
+from search.elasticsearch_helpers import label_autocomplete
+
+
+@registry.register_document
+class SchoolDocument(Document):
+
+    autocomplete = fields.TextField(analyzer=label_autocomplete)
+    text = fields.TextField(attr='settlement_school')
```

Comment:
We should not be indexing the "settlement_school" field. This is for internal use and would server no search purpose.

The things that would be useful for search are:
- primary_alias (a property that needs to called in via "prepare" on an instance, since it's not a db column
- pk
- city
- state
- zip5

Reply:
Added zip5 and changed settlement_school to primary_alias